src/DTO/Reaction.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace DefStudio\Telegraph\DTO;
+
+use Illuminate\Contracts\Support\Arrayable;
+
+/**
+ * @implements Arrayable<string, string>
+ */
+class Reaction implements Arrayable
+{
+    private string $type;
+    private ?string $emoji = null;
+    private ?string $customEmojiId = null;
+
+    private function __construct()
```

Comment:
An empty method can be deleted. There is no point in keeping it.

Reply:
It's not just empty without purpose—making. Private constructor enforces controlled object creation. This prevents direct instantiation and ensures that objects can only be created through specific methods, giving more control over how and when instances are generated.